gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
+        python_path = 'C:\\Python27'
+        nsis_path = 'C:\\NSIS'
+        if options.python_path:
+            python_path = options.python_path
+        if options.nsis_path:
+            nsis_path = options.nsis_pat
     python_lib_path = os.path.join(
-        os.getenv('MOZREGUI_PYTHONPATH', "C:\\Python27"),
+        os.getenv('MOZREGUI_PYTHONPATH', python_path),
         "Lib"
-    )
+        )
     makensis_path = os.path.join(
-        os.getenv('MOZREGUI_NSISPATH', "C:\\NSIS"),
+        os.getenv('MOZREGUI_NSISPATH', nsis_path),
         "makensis.exe"
-    )
+        )
```

Comment:
same thing here - let the parenthesis where they were, and please remove the os.getenv call.